tests/roles/ovn_adoption/tasks/main.yaml

```diff
@@ -148,9 +148,9 @@
     {{ oc_header }}
     {{ ovn_copy_shell_vars }}
 
-    $CONTROLLER1_SSH sudo systemctl stop tripleo_ovn_cluster_northd.service
-    $CONTROLLER2_SSH sudo systemctl stop tripleo_ovn_cluster_northd.service
-    $CONTROLLER3_SSH sudo systemctl stop tripleo_ovn_cluster_northd.service
+    $CONTROLLER1_SSH if sudo systemctl is-active tripleo_ovn_cluster_northd.service ';' then sudo systemctl stop tripleo_ovn_cluster_northd.service ';' fi
```

Comment:
I'm a bit surprised this works (running bash expressions like this rather than just a simple command), i need to do some experimentation to verify this indeed does what we think it does... :)

Reply:
I tested this long ago, it works. We are using the similar tricks with must-gather and sos reports